packages/@magic-ext/react-native-bare-oauth/package.json

```diff
@@ -35,6 +35,7 @@
     "@types/crypto-js": "~3.1.47"
   },
   "peerDependencies": {
-    "@magic-sdk/react-native-bare": ">=13.0.0"
+    "@magic-sdk/react-native-bare": ">=13.0.0",
+    "react-native-inappbrowser-reborn": ">=3.7.0"
```

Comment:
shall we move this to devDependencies, as it's already peer?

https://github.com/magiclabs/magic-js/pull/432/files#diff-00b1eb415b6404cff50cddee27041bbc8742807d50be7d5ced8916f78236dda2L31

Reply:
Good catch!